Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -600,8 +600,8 @@ private async Task<bool> CanUpdateAsync(string resource)
                             if (newsItem.Accepted != false)
                                 return false;
 
-                            // can update if user is admin
-                            if (user_groups.Contains(AuthGroup.SiteAdmin))
+                            // can update if user is admin or SNAdmin
+                            if (user_groups.Contains(AuthGroup.SiteAdmin) || user_groups.Contains(AuthGroup.NewsAdmin))
```

Comment:
```suggestion
                            if (user_groups.Contains(AuthGroup.NewsAdmin))
```
Just trying to get away from giving site admins permissions that are not necessarily theirs to have.